### DISCUSSION TITLE
Speed extremely limited just like a normal VPN provider

### DISCUSSION THREAD

**Discussion by SiteSplat** [OutlineFoundation/outline-sdk#77] [General]
I just setup outline in a local server just 1h away from my location.

The speed test was kinda underwhelming... 750mbs - (I have a 8gbs line.)
That download speed is the same as a normal VPN provider.
The ping was also higher compared to a normal VPN. 6.5ms vs 5ms.

What am I missing? 
isn't the purpose of running  a local VPN to have raw speed as well?
That is what's advertised from DO email by the way...

**Comment by jyyi1** [comment]
Hi @SiteSplat , thanks for reporting this. We might need more detailed information in order to resolve this performance issue. For example, what was the download/upload bandwidth of your server? What is the latency to connect to the server (sometimes physical distance doesn't matter)? And also what operating system are you using?

Please note that this Discussion section is focused on developers using Outline SDK. For this issue, I would recommend you to submit an issue in [outline-client repository](https://github.com/Jigsaw-Code/outline-client/issues). Thanks.

**Comment by fortuna** [comment]
@SiteSplat How did you do the test?
How did you deploy the server?
Are you using the Outline app or the SDK?
